modules/deploy/pages/deployment-option/cloud/security/cloud-authentication.adoc

```diff
@@ -19,7 +19,7 @@ Redpanda Cloud can authenticate users with single sign-on (SSO) to an OIDC-based
 
 ==== Integrate IdP
 
-You must integrate your IdP with Redpanda Cloud to use SSO. On the *Users* page, you can add connections for up to two different IdPs. Enter the client ID, client secret, and discovery URI for the IdP. (See your IdP documentation for these values. The discovery URI may be called something different, like the well known URL or the `issuer_url`.) 
```

Comment:
> you can add connections for up to two different IdPs

It's worth telling customers that if they have a use case which requires more, they should contact us/let us know.

Reply:
@deniscoady Do you want the docs to prompt users for additional IdP use cases/values? I'm a bit torn, as docs is not typically  where customers make such requests. Please advise.

Reply:
We plan to call this out in the product so we probably shouldn't make a call to action in the docs.